### ISSUE TITLE
Training results are different

### ISSUE THREAD

**Issue by lidaxia223** [albrateanu/LYT-Net#40]
So what are the metrics for training with the source code? Why did my metrics improve only in the first 100 epochs, but stop improving after the last 1400 epochs? My PSNR on LOLv1 using PyTorch and TensorFlow frameworks is around 25.5, which seems to indicate that the training process doesn't significantly improve the pre-trained model.

**Comment by lidaxia223** [comment]
I found that after disabling GT-mean, the PSNR metric on PyTorch only reaches 21, while on TensorFlow it only reaches 22.
